db/migrate/20220318210131_create_testimonials.rb

```diff
@@ -0,0 +1,10 @@
+class CreateTestimonials < ActiveRecord::Migration[6.1]
+  def change
+    create_table :testimonials do |t|
+      t.string :name, NULL: false
```

Comment:
Just for the sake of respecting rails conventions, don't use uppercase words here.
```suggestion
      t.string :name, null: false
```